.buildkite/smart_exhaustive_tests_pipeline.yml

```diff
@@ -16,19 +16,16 @@ steps:
                   commit: "${BUILDKITE_COMMIT}"
                   branch: "${BUILDKITE_BRANCH}"
                   env:
-                    - BUILDKITE_PULL_REQUEST=${BUILDKITE_PULL_REQUEST}
-                    - BUILDKITE_PULL_REQUEST_BASE_BRANCH=${BUILDKITE_PULL_REQUEST_BASE_BRANCH}
-                    - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
-                    - ELASTIC_SLACK_NOTIFICATIONS_ENABLE=false
+                    BUILDKITE_PULL_REQUEST: "${BUILDKITE_PULL_REQUEST}"
+                    BUILDKITE_PULL_REQUEST_BASE_BRANCH: "${BUILDKITE_PULL_REQUEST_BASE_BRANCH}"
```

Comment:
`reviewer`: it requires this format, although, I tried to use the same as we have in other places, like `beats` and didn't work.